ietf/utils/templates/blocks/note_well_block.html

```diff
@@ -1,9 +1,32 @@
-{% if note_well.text %}
-    <div>{{ note_well.text|safe }}</div>
-{% else %}
-    {# Fall back to error text #}
+<div id="notewell_text">
     <h2>Note Well</h2>
     <p>An error occured fetching the note well.</p>
     <p>See <a href="https://github.com/ietf/note-well/">https://github.com/ietf/note-well/</a> for the note well.</p>   
     <p>Please <a href="mailto:support@ietf.org">email</a> us to report this.</p>
-{% endif %}
\ No newline at end of file
+</div>
+<script 
+    src="https://cdn.jsdelivr.net/npm/showdown@2.1.0/dist/showdown.min.js
+"></script>
+<script>
+    async function setNotewellHTML() {
+        const response = await fetch (
+            'https://raw.githubusercontent.com/ietf/note-well/65ea9057c9e93c027bc4b96183c47b0a8c1c17ea/note-well.md'
```

Comment:
Can you make this configurable via settings?
There's a settings entry already for this.
See https://github.com/ietf-tools/wagtail_website/blob/79e0abc996cbce120a6b9d3b99db90150ba7520c/ietf/settings/base.py#LL247C1-L247C75